user.css

```diff
@@ -1732,65 +1757,68 @@ option {
-  -webkit-mask-image: url('https://cdn.jsdelivr.net/gh/nimsandu/spicetify-bloom@master/assets/fluentui-system-icons/ic_fluent_skip_forward_15_24_filled.svg');
-  background-color: var(--spice-text) !important;
+  -webkit-mask-image: url('./fluentui-system-icons/ic_fluent_skip_forward_15_24_filled.svg');
+  background-color: var(--spice-text) !important; 
   height: 22px !important;
 }
 
 [aria-label="Change speed"] { /* Speed icon for podcasts */
-  -webkit-mask-image: url('https://cdn.jsdelivr.net/gh/nimsandu/spicetify-bloom@master/assets/fluentui-system-icons/ic_fluent_top_speed_24_filled.svg');
-  background-color: var(--spice-text) !important;
+  -webkit-mask-image: url('./fluentui-system-icons/ic_fluent_top_speed_24_filled.svg');
+  background-color: var(--spice-text) !important; 
   height: 22px !important;
 }
 
 /* Disabled */
 .main-repeatButton-button[aria-checked="false"]{
-  -webkit-mask-image: url('https://cdn.jsdelivr.net/gh/nimsandu/spicetify-bloom@master/assets/fluentui-system-icons/ic_fluent_arrow_repeat_all_off_24_regular.svg');
+  -webkit-mask-image: url('./fluentui-system-icons/ic_fluent_arrow_repeat_all_off_24_regular.svg');
   background-color: var(--spice-subtext) !important;
-  height: 22px !important;
+  height: 23px !important; 
+  
 }
 /* Repeat one */
 .main-repeatButton-button[aria-checked="mixed"]{
-  -webkit-mask-image: url('https://cdn.jsdelivr.net/gh/nimsandu/spicetify-bloom@master/assets/fluentui-system-icons/ic_fluent_arrow_repeat_all_once_24_filled.svg') !important;
-  background-color: var(--spice-text) !important;
+  -webkit-mask-image: url('./fluentui-system-icons/ic_fluent_arrow_repeat_all_once_24_filled.svg') !important;
+  background-color: #00ffa1 !important;
   height: 22px !important;
-}
+  
+color: #00ffa1 !important;}
 /* Repeat all */
 .main-repeatButton-button[aria-checked="true"]{
-  -webkit-mask-image: url('https://cdn.jsdelivr.net/gh/nimsandu/spicetify-bloom@master/assets/fluentui-system-icons/ic_fluent_arrow_repeat_all_24_filled.svg');
-  background-color: var(--spice-text) !important;
-  height: 22px !important;
+  -webkit-mask-image: url('./fluentui-system-icons/ic_fluent_arrow_repeat_all_24_filled.svg');
+  background-color: #00ffa1 !important;
+  color: #00ffa1 !important;height: 22px !important;
+  
 }
 
 /* Shuffle disabled */
 .main-shuffleButton-button {
-  -webkit-mask-image: url('https://cdn.jsdelivr.net/gh/nimsandu/spicetify-bloom@master/assets/phosphor-icons/shuffle.svg');
+  -webkit-mask-image: url('./phosphor-icons/shuffle.svg');
   background-color: var(--spice-subtext) !important;
-  height: 22px !important;
+  height: 23px !important;
 }
 /* Shuffle enabled */
-.main-shuffleButton-button[aria-checked="true"] {
-  -webkit-mask-image: url('https://cdn.jsdelivr.net/gh/nimsandu/spicetify-bloom@master/assets/phosphor-icons/shuffle-bold.svg');
-  background-color: var(--spice-text) !important;
-  height: 22px !important;
+.main-shuffleButton-button.OF_3F0SQCsBtL1jSTlTA {
```

Comment:
Refrain from using hashes as class names when there are better alternatives like the current one as those can be easily broken in the next updates of Spotify.